### DISCUSSION TITLE
Add a new repo on Renovate Github app (programmatic)

### DISCUSSION THREAD

**Discussion by DanielCastronovo** [renovatebot/renovate#29990] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I would like to add a new repository on Renovate, with Github API GraphQL, or other way to do it programmatic.
Currently update the list of repository is not supported.
We need to export the actual list in a file, modify it and push it with the new repo with the Github API GraphQL

**Comment by RahulGautamSingh** [comment]
https://docs.github.com/en/rest/apps/installations?apiVersion=2022-11-28#add-a-repository-to-an-app-installation
Can only be added or removed one at a time.

**Comment by DanielCastronovo** [comment]
Yes, so it could be better to update it

**Comment by DanielCastronovo** [comment]
Any news ? because it difficult to manage this manualy

**Reply by rarkins** [reply]
Aren't you asking the wrong company? It sounds like you're asking about GitHub's limitation, something which isn't specific to the Renovate app
